src/pykeen/nn/modules.py

```diff
@@ -1704,6 +1738,245 @@ def _prepare_state_for_functional(self) -> MutableMapping[str, Any]:  # noqa: D1
         state["tanh_map"] = self.tanh_map
         return state
 
+    @classmethod
+    def product_normalize(cls, x: torch.FloatTensor, dim: int = -1) -> torch.FloatTensor:
```

Comment:
static method?

Reply:
if the class isn't referenced itself that's the best time to use staticmethod instead of classmethod. this is proabbly the case for other parts of this PR

Reply:
done in e0c5ae9